src/routes/+page.svelte

```diff
@@ -100,40 +93,40 @@
       <h1><i class="fa-solid fa-file-lines"></i> MegaResume</h1>
     </header>
     <ul>
-      <a href="#import">
+      <a on:click={onMenuClick} href="#import">
```

Comment:
I thought about the best way to do this, and I decided this is the best way. Although it seems repetitive to assign this `on:click` function, the number of options in the menu will not change often, and it's easier to assign this function to each of these links than observing the changes in either the workStore or the markup, and then assigning the click handler on a change.